tests/std/tests/VSO_0000000_regex_use/test.cpp

```diff
@@ -546,6 +546,17 @@ void test_VSO_226914_word_boundaries() {
     aWordAny.should_search_fail("aa", match_not_bow | match_not_eow);
 }
 
+void test_gh_731() {
+    std::vector<std::string> groups{"AAA", "BBB", ""};

```

Comment:
Should probably include `<vector>` even if `<test_regex_support.hpp>` already does so.